src/features/slackapi/slackapi-utils.ts

```diff
@@ -88,17 +101,43 @@ Have a great rest of the day!
       billableHoursPercentage
     } = TimeUtilities.calculateWorkedTimeAndBillableHours(user.selectedWeek);
 
-    return `
-Hi ${name},
+    const customMessage = `
+Hi ${firstName},
 Last week (week: ${ week }, ${startDate} - ${endDate}) you worked ${displayLogged} with an expected time of ${displayExpected}.
 ${message}
 Project time: ${displayProject}, Internal time: ${displayInternal}.
 Your percentage of billable hours was: ${billableHoursPercentage}%
 You ${+billableHoursPercentage >= 75 ? "worked the target 75% billable hours last week:+1:" : "did not work the target 75% billable hours last week:-1:"}.
 Have a great week!
     `;
+    return {
+      message: customMessage,
+      name: name,
+      week: week,
+      startDate: startDate,
+      endDate: endDate,
+      displayLogged: displayLogged,
+      displayExpected: displayExpected,
+      displayProject: displayProject,
+      displayInternal: displayInternal,
+      billableHoursPercentage: billableHoursPercentage
+    };
```

Comment:
I'd create an own interface for this so you can make sure that the typing is correct everywhere where this data is used.